src/components/BoardGenerator/BoardGenerator.js

```diff
@@ -0,0 +1,29 @@
+import { v4 as uuidv4 } from 'uuid';
+import { animals, frozen } from '../../componentAssets/Images';
+
+const BoardGenerator = (cardBack, boardSize) => {
+  const cardBackImg = cardBack === 'animals' ? animals : frozen;
+  const createArray = (boardSize) => {
+    const animalsObj = Object.keys(cardBackImg);
+    const singleArray = Array.from({ length: boardSize }, () => ({}));
+    singleArray.forEach((el, idx) => {
+      el.imgSrc = cardBackImg[animalsObj[idx]];
+      el.pairId = idx;
+      el.isVisible = false;
+    });
```

Comment:
We could probably simplify it like: 
`const singleArray = Array.from(Array(boardSize)).map((el, idx) => {return {imgSrc: cardBackImg[animalsObj[idx]], pairId = idx, isVisible = false});`

Not sure if it is simplifying :D

Reply:
Sure its simplifying but its less readable, I would like to leave it that way.